for_loop_practice.py

```diff
@@ -0,0 +1,31 @@
+questions = ["What is 1 + 1?", "What is 2 + 2?"]
+answers = [2, 4]
+
+for question in questions:
+    print(question)
+    user_answer = input("Type answer and press enter ==> ")
+    user_answer = int(user_answer)
+    if user_answer == int(answers[0]) or user_answer == answers[1]:
+        print("Well done!")

```

Comment:
You should check if the answer is equal to the answer of the correct question. Say I add 5 more questions, I would need to update this condition. You could do something like this
```
for i in range(0, len(questions)): # len(questions) will be the number of questions
    current_question = questions[i]
    current_answer = answers[i]
    # and then you can compare current answer and user answer
```

Reply:
I like that